packages/jest-environment-jsdom/src/index.ts

```diff
@@ -45,6 +45,9 @@ class JSDOMEnvironment implements JestEnvironment {
       throw new Error('JSDOM did not return a Window object');
     }
 
+    // for "universal" code (code should use `globalThis`)
+    global.global = global;
```

Comment:
the need for this makes it a breaking change as environments haven't had to do this before